interview-questions.md

```diff
@@ -8,50 +8,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: A conditional statment is a way to make decisions by controling the flow of outcome based off of evaluations of imput.  
 
-Researched answer:
+Researched answer: Conditional statments use if/else statments and the evaluations check boolean true or false values.  The comparison of condtions can be made using equality, logical, and relational operations.
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is a version control software that allows developers to manage files, work flow, and documentation of repositories on local computers.  It also allows for independent contributions to be made to workspaces and projects that can be compiled to an online directory.
 
-Researched answer:
+GitHub is the remote or online terminal or nexus where repositiories of multi faceted projects are dispached, retrieved, compiled and managed from.  The files live on a cloud which can be acceess remotly and imported locally to individual computers.
+
+Researched answer: Git is an opensource DevOps tool that allows for remote management and tracking of source code.  It allows for non linear contributions to be made by multiple developers.  
+
+GitHub is a code hosting platform for version control of files and colaboration of work between multiple contributor can centrally located.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: Equality, Logical, Relational operators will all return an evaluation of a boolean data type.
+
+Researched answer: the ! or bang boolean operator is evaluating if condition is the opposite of what is being compared.  It is not returning the direct true or false boolean logical value of the condition being evaluated.
```

Comment:
Great job talking about something you hadn't mentioned in your initial answer! 👏🏽